packages/react/src/components/tabs/tab-panel.tsx

```diff
@@ -5,35 +5,42 @@ import { focus, focusVisibleReset } from '../../utils/css-state';
 interface TabPanelProps {
     buttonId: string,
     children: ReactNode;
-    contained?: boolean;
     hidden: boolean,
     id: string,
+    global?: boolean;
 }
 
-const StyledDiv = styled.div<{ $contained?: boolean }>`
-    border: ${({ $contained, theme }) => ($contained ? `1px solid ${theme.component['tab-panel-border-color']}` : 'none')};
+const StyledDiv = styled.div<{ $isGlobal?: boolean; }>`
+    background: ${({ $isGlobal, theme }) => !$isGlobal && theme.component['tabs-panel-background-color']};
+    border: ${({ $isGlobal, theme }) => !$isGlobal && `1px solid ${theme.component['tabs-panel-border-color']}`};
+    border-radius: ${({ $isGlobal }) => !$isGlobal && '0 0 var(--border-radius-2x) var(--border-radius-2x)'};
     border-top: none;
 
-    ${focus}
-    ${({ theme }) => focus({ theme }, false, ':focus-visible')}
-    ${focusVisibleReset}
+    ${focus};
+    ${focusVisibleReset};
+
+    &:focus-visible {
+        position: relative;
+        z-index: 2;
+    }
 `;
 
 export const TabPanel: FunctionComponent<PropsWithChildren<TabPanelProps>> = ({
     buttonId,
     children,
-    contained,
     hidden,
     id,
+    global,
```

Comment:
Valeur par défaut? Aussi tu peux le remonter avant `hidden` pour garder l'ordre alpha
```suggestion
    global = false,
```